assets/css/guides.less

```diff
@@ -621,14 +621,14 @@ body .guides .guides-section-white {
 }
 
 .fixed {
-	position: fixed;
-	top: 0;
+	position: fixed !important;
+	top: 0 !important;
 }
 
 .bottom {
-	position: absolute;
-	bottom: 0;
-	top: auto;
+	position: absolute !important;
+    bottom: 0;
+    top: auto !important
```

Comment:
Why do these all need `!important`?

Reply:
I had to use important because the new position I specified on page load gets retained even when it should be picking up the fixed and bottom styles.